myapp/views.py

```diff
@@ -0,0 +1,24 @@
+from rest_framework import generics
+from .models import Author, Post
+from .serializers import AuthorSerializer, PostSerializer
+
+
+# Create your views here.
+class AuthorListCreateView(generics.ListCreateAPIView):
+    queryset = Author.objects.all()
+    serializer_class = AuthorSerializer
+
+
+class AuthorDetailView(generics.RetrieveUpdateDestroyAPIView):
+    queryset = Author.objects.all()
+    serializer_class = AuthorSerializer
```

Comment:
Can you take a stab at flipping from 2 APIViews per model to ViewSets

Reply:
Sure thing.... changes implemented to use one ViewSet per model. Please confirm.

Reply:
Also modified the tests to accommodate the changes

Reply:
updates look good, thanks

Reply:
Thank you boss. Looking forward to hear on the next steps :)